get_together/settings.py

```diff
@@ -209,6 +210,26 @@
     messages.ERROR: "alert-danger",
 }
 
+THEME_CONFIG = {
+    "actions": {
+        "host_event": {
+            "label": _("Host a Get Together"),
+            "icon": "far fa-calendar-plus",
+        }
+    },
+    "text_labels": {
+        "host": _("host"),
```

Comment:
Please change "host" to "Host" so that the text matches the previous text by default.